lib/node_modules/@stdlib/math/base/special/sincos/lib/kernel_sincos.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+/*
+* The following copyright, license, and long comment were part of the original implementation available as part of [FreeBSD]{@link https://svnweb.freebsd.org/base/release/9.3.0/lib/msun/src/k_sin.c?view=co}.
+*
+* The implementation follows the original, but has been modified for JavaScript.
+*/
+
+/*
+* ====================================================
+* Copyright (C) 1993 by Sun Microsystems, Inc. All rights reserved.
+*
+* Developed at SunSoft, a Sun Microsystems, Inc. business.
+* Permission to use, copy, modify, and distribute this
+* software is freely granted, provided that this notice
+* is preserved.
+* ====================================================
+*/
+
+// VARIABLES //
+
+var S1 = -1.66666666666666324348e-01; // 0xBFC55555, 0x55555549
+var S2 = 8.33333333332248946124e-03;  // 0x3F811111, 0x1110F8A6
+var S3 = -1.98412698298579493134e-04; // 0xBF2A01A0, 0x19C161D5
+var S4 = 2.75573137070700676789e-06;  // 0x3EC71DE3, 0x57B1FE7D
+var S5 = -2.50507602534068634195e-08; // 0xBE5AE5E6, 0x8A2B9CEB
+var S6 = 1.58969099521155010221e-10;  // 0x3DE5D93A, 0x5ACFD57C
+
+var C1 = 4.16666666666666019037e-02;  // 0x3FA55555, 0x5555554C
+var C2 = -1.38888888888741095749e-03; // 0xBF56C16C, 0x16C15177
+var C3 = 2.48015872894767294178e-05;  // 0x3EFA01A0, 0x19CB1590
+var C4 = -2.75573143513906633035e-07; // 0xBE927E4F, 0x809C52AD
+var C5 = 2.08757232129817482790e-09;  // 0x3E21EE9E, 0xBDB4B1C4
+var C6 = -1.13596475577881948265e-11; // 0xBDA8FAE9, 0xBE8838D4
+
+
+// MAIN //
+
+/**
+* Computes the sine and cosine on \\( \approx [-\pi/4, \pi/4] \\) (except on \\(-0\\)), where \\( \pi/4 \approx 0.7854 \\).
+*
+* @private
+* @param {array} out - destination array
```

Comment:
Okay, I've attempted to fix this in the source files. They are now to the best of my knowledge correct.